i18n/locales/en.json

```diff
@@ -98,6 +98,12 @@
         "status": "Status",
         "modified": "Modified",
         "submitted": "Submitted"
+    },
+    "moderateSubmissionForm": {
+        "contactInformation": "Contact Information",
+        "addresses": "Addresses",
+        "googleMapsInformation": "Google maps information",
+        "healthcareProfessionalId": "Healthcare Professional IDs"
```

Comment:
Sorry I forgot to mention this before, but let's add these strings to ja.json as well. They don't need to be translated yet, but we at least want them there so they'll be included in the next round of translations.